### ISSUE TITLE
multicoil-dataset

### ISSUE THREAD

**Issue by chke097** [utcsilab/deep-jsense#1]
Hi, Could you please share the dataset of the article? or can you tell us exactly which data sets are being used?

**Comment by mariusarvinte** [comment]
Hello, thanks for your question. We used multi-coil knee from the fastMRI dataset: https://fastmri.org/dataset/. This is a dataset hosted by NYU, see here for instructions on how to download it: https://fastmri.med.nyu.edu/.

More specifically, we used all volumes from `knee_multicoil_train.tar.gz` (this includes both fat suppressed and non-fat suppressed multi-slice knee scans), and from each volume we selected slices {13, 14, 15, 16, 17}. 

After you download and extract the files, you'd have to also estimate sensitivity maps. We did this by using the ESPIRiT algorithm from the BART toolbox (see here: https://bart-doc.readthedocs.io/en/latest/tools.html), but anything else should work.

Once you estimate and save the maps to files with matching filenames as the fastMRI dataset, the code in https://github.com/utcsilab/deep-jsense/blob/main/train.py#L36-L44 can be modified to point to the correct folders, after which the rest of the script should work.

**Comment by chke097** [comment]
@mariusarvinte Hello, I am very interested in this work you did. I tried to estimate coil sensitivity maps according to your instructions, but all failed. So, would you like to share the source code to estimate the coil sensitivity maps? Thank you!

**Comment by mariusarvinte** [comment]
Thanks for the follow-up, I have uploaded the exact script we used: https://github.com/utcsilab/deep-jsense/blob/main/estimate_maps.py

To run this, you will first need to install BART v0.8.0 following the instructions here: https://mrirecon.github.io/bart/installation.html.

Then, you will also need the fastMRI dataset or a portion of it downloaded and unzipped in your local PC. The file structure should be that of the default fastMRI: A single folder with multiple .h5 files in it.

Lastly, you would need to edit the following in `estimate_maps.py` before running:
- https://github.com/utcsilab/deep-jsense/blob/0ef8927b77962334706af32fa78d5d176efe1c4c/estimate_maps.py#L7-L8 with the path to your local BART folder.
Note the explicit `python` sub-folder of BART in the second line, that is required there.
- https://github.com/utcsilab/deep-jsense/blob/3a5edd0226c030dd2f62edff1482c77367b57eb6/estimate_maps.py#L17 with the string variable pointing to the fastMRI data folder you wish to estimate sensitivity maps for.

To finally run sensitivity map estimation after the above a simple call `python estimate_maps.py` will do it.

Let me know if this was helpful.
